assets/src/edit-story/components/form/select.js

```diff
@@ -43,10 +70,12 @@ function SelectMenu({
   onChange,
   postfix,
   disabled,
+  flexGrow,
+  flexBasis,
 }) {
   return (
-    <Group disabled={disabled}>
-      <Label>{label}</Label>
+    <Container disabled={disabled} flexBasis={flexBasis} flexGrow={flexGrow}>
+      {label && <Label>{label}</Label>}
```

Comment:
If `label` is optional, the prop types definition should be adjusted accordingly (currently marked as `isRequired`.

Reply:
PropTypes adjusted